### PR TITLE
Add page for single-cell specific resources

### LINKED ISSUE #18: Nice to have: page of advanced single-cell related resources
Here's what we have for RRP: https://alexslemonade.github.io/2022-yi-workshops/reproducibility-resources.html

We'd likely want to include things like cell typing methods we won't cover but people might want to know about.

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #18 

Here I added an additional page for single-cell specific resources. I modeled this after the `Additional Resources` page from the RRP site since this was linked in the issue. I chose to include links to relevant tutorials and vignettes as well as any relevant papers for some common topics for single-cell that might be of interest. 

A few questions: 

- Are we okay with this style of listing resources? Do we want to add anymore context here? 
- Are there things I'm missing or should not include? 
- We still have a separate page for consultation sessions that includes different resources and I chose to keep this separate. I think that's what we want since that page has things like scpca on it whereas I see this being links to guides/ tutorials/ papers to help complement our instruction. 

The only section I haven't fully filled out yet is on differential abundance, because I need to dig a little deeper to decide what I want to put there. But I wanted to file this to get some initial thoughts and answers to these questions to at least make sure formatting was how we wanted it.

### CODE REVIEW

**Review by jashapiro — COMMENTED** [review]
Overall I think this looks good, but I'd update citations to remove journal names... Maybe add authors and years instead.  Because I am anti-journal like that.

A couple cell typing links that might be worth linking:
One is the 10X page about cell typing: https://www.10xgenomics.com/resources/analysis-
guides/web-resources-for-cell-type-annotation

Then maybe a review paper? http://dx.doi.org/10.1093/bfgp/elaa003 and/or https://doi.org/10.1016/j.csbj.2021.10.027
(I'm a bit less enamored of linking to these papers, but see what you think)

File: workshop/single-cell-resources.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+title: Resources for single-cell RNA-seq analysis
+nav_title: Single-cell resources
+---
+
+This list provides some links to resources on single-cell RNA-seq analysis methods that may be useful to you as you develop your own single-cell RNA-seq analysis skills and practices.
+Please note, this is not an exhaustive list.
+It includes multiple types of resources for various topics in single-cell RNA-seq analysis, but does not represent the complete breadth of analysis topics.
+Resources are listed by topic and in alphabetical order, not in order of recommendation.
+
+**Table of contents**
```
Comment (sjspielman) [inline workshop/single-cell-resources.md:11]: Do we want to keep the doctoc comments here for future updates, as in - https://github.com/AlexsLemonade/training-specific-template/issues/98

File: workshop/single-cell-resources.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- [AUCell: Identifying cells with active gene sets](https://bioconductor.org/packages/devel/bioc/vignettes/AUCell/inst/doc/AUCell.html)
+- [Azimuth](https://azimuth.hubmapconsortium.org/)
+- [Fully-automated and ultra-fast cell-type identification using specific marker combinations from single-cell transcriptomic data - _Nature Communications_](https://doi.org/10.1038/s41467-022-28803-w)
+- [OSCA chapter on cell type annotation](https://bioconductor.org/books/3.16/OSCA.basic/cell-type-annotation.html)
+- [scType](http://session.asuscomm.com/)
+- [The SingleR Book - Bioconductor](https://bioconductor.org/books/3.16/SingleRBook/)
+
+## CITE-seq
+
+- [OSCA chapter on integration with protein abundance](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html)
```
Comment (sjspielman) [inline workshop/single-cell-resources.md:63]:
```suggestion
- [OSCA chapter on integrating with protein abundance](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html)
```

**Comment by allyhawkins** [comment]
Okay I updated the references to remove the journals and include the authors and year. I formatted them like we do with in text citations, but I can change that if others have citation opinions. 

I also added a few more references, some Josh mentioned, another one for DE that I like, and also some for differential abundance. I tried to find a good tutorial or vignette on DA, but the best one is the OSCA one and even then... meh. So I mostly included papers - one comparison of methods paper on biorxiv, and then two methods papers - one of which is from the scvi group that I had looked into before that I like. This is ready for another round of review.

**Review by jashapiro — APPROVED** [review]
LGTM! I had a thought about going through and changing a bunch of en-dashes to em-dashes, but I will refrain.

File: workshop/single-cell-resources.md (hunk 1 above)
Comment (sjspielman) [reply workshop/single-cell-resources.md:11]: Still wondering about this comment..?

**Review by sjspielman — APPROVED** [review]

File: workshop/single-cell-resources.md (hunk 1 above)
Comment (allyhawkins) [reply workshop/single-cell-resources.md:11]: Ah sorry I meant to address this, I'll add in doctoc now.

File: workshop/single-cell-resources.md [hunk 3]
```diff
@@ -8,7 +8,9 @@ Please note, this is not an exhaustive list.
 It includes multiple types of resources for various topics in single-cell RNA-seq analysis, but does not represent the complete breadth of analysis topics.
 Resources are listed by topic and in alphabetical order, not in order of recommendation.
 
-**Table of contents**
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```
Comment (jashapiro) [inline workshop/single-cell-resources.md:13]:
```suggestion
**Table of Contents** 
```

Comment (sjspielman) [reply workshop/single-cell-resources.md:13]: Just peeked at this, and we can do - https://github.com/thlorenz/doctoc#specifying-a-custom-toc-title

tested locally, and it needs the `**` - 
```
doctoc blah.md --title **Table of Contents**
```

Comment (allyhawkins) [reply workshop/single-cell-resources.md]: Thanks Stephanie! Should be fixed now using that trick.

Comment (jashapiro) [reply workshop/single-cell-resources.md]: Once it is removed, it stays removed, so either way would have been fine!

**Pull request merged**